aws/cloudformation/ecs/posthog.yaml

```diff
@@ -105,48 +120,128 @@ Parameters:
                  access other AWS resources like S3 buckets, DynamoDB tables, etc
   RDSNodeType:
     Description: RDS node instance class, e.g. t3.micro(free tier). See https://aws.amazon.com/rds/instance-types/
+                 If you're just testing PostHog out, the T3 is fine. If you want to run PostHog in production,
+                 you should at least run an M5. The new Graviton2 instances (M6g and R6g) are great in that they
```

Comment:
I don't think we support ARM in production yet

Reply:
We've been running Postgres on an ARM instance for 2 months now with no issue.

Reply:
Ah, right, this is RDS, so should definitely be food with ARM 👍